docs/snippets/getting-started/anta_nrfu_table.output

```diff
@@ -0,0 +1,45 @@
+                    Total number of selected tests: 45
+                    Maximum number of open file descriptors for the current ANTA process: 16384
+                    ---------------------------------
+           INFO     Preparing ANTA NRFU Run completed in: 0:00:00.069.                                                                                                                                                                                                                                  tools.py:302
+           INFO     Running ANTA tests ...                                                                                                                                                                                                                                                              tools.py:294
+[10:29:09] INFO     Running ANTA tests completed in: 0:00:00.936.                                                                                                                                                                                                                                       tools.py:302
+           INFO     Cache statistics for 's1-spine1': 1 hits / 9 command(s) (11.11%)                                                                                                                                                                                                                    runner.py:75
+           INFO     Cache statistics for 's1-spine2': 1 hits / 9 command(s) (11.11%)                                                                                                                                                                                                                    runner.py:75
+           INFO     Cache statistics for 's1-leaf1': 1 hits / 9 command(s) (11.11%)                                                                                                                                                                                                                     runner.py:75
+           INFO     Cache statistics for 's1-leaf2': 1 hits / 9 command(s) (11.11%)                                                                                                                                                                                                                     runner.py:75
+           INFO     Cache statistics for 's1-leaf3': 1 hits / 9 command(s) (11.11%)                                                                                                                                                                                                                     runner.py:75
+  • Running NRFU Tests...100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 45/45 • 0:00:00 • 0:00:00
```

Comment:
Not sure why but on my screen the table is way too large with wasted space in the `Message(s)` section. There is also a scroll bar at the bottom which is unnecessary if that space was not wasted:
![image](https://github.com/user-attachments/assets/69052a83-857a-458e-b7f4-21093a4c33a9)

Same behavior for the text and JSON outputs.

Reply:
should be fixed.. re ran in ATD with COLUMNS set to 160.. it was .. stupid